maintainers/maintainer-list.nix

```diff
@@ -9734,4 +9734,14 @@
     github = "yanganto";
     githubId = 10803111;
   };
+  starcraft66 = {
+    name = "Tristan Gosselin-Hane";
+    email = "starcraft66@gmail.com";
+    github = "starcraft66";
+    githubId = 1858154;
+    keys = [{
+      "rsa4096/0x9D98CDACFF04FD78"
+      "8597 4506 EC69 5392 0443  0805 9D98 CDAC FF04 FD78"
```

Comment:
This is a syntax error since an attribute-set always requires a name for each value.
In this case you'd want to write sth. like `{ longkeyid = "rsa4096..."; fingerprint = "8597 ..."; }`.